action.yml

```diff
@@ -9,6 +9,8 @@ inputs:
     description: Deployment environment.
     required: true
 outputs:
+  deployAllPackages:
```

Comment:
I don't see this flag added by any of the code changed in this PR. Was it just forgotten from earlier?